### DISCUSSION TITLE
What is the purpose of the stats server running on 23233?

### DISCUSSION THREAD

**Discussion by bw9ubwo** [charmbracelet/soft-serve#590] [Q&A]
I understand that it is also possible to serve git repositories over HTTP or GIT. But what does the server running on 23233? The only information is "The stats server configuration." from the config file.

**Comment by aymanbagabas** [comment]
Hi @bw9ubwo, glad you brought this up. It's a Prometheus stats server that can be used to scrap metrics about Soft Serve

**Comment by bw9ubwo** [comment]
Thanks! Personal I don't use the http, git and stats part, so I would like to see if I could switch them off in the config.

**Reply by aymanbagabas** [reply]
I think that would be a great addition to Soft Serve. Could you please open a new feature request for this one?
